### ISSUE TITLE
Data values flashing for a split second then disappearing.

### ISSUE THREAD

**Issue by saturnaut** [msavin/Mongol#90]
Just ran a Meteor update. Mongol opens fine, no errors in either console. Clicking a collection shows the right number of entries, and I can cycle through them. But as each collection entry loads, it flashes the correct data for a split second, then the values disappear.

The fields I defined in the collection are still there, and even entries that have optional fields only show the field when that data was defined. But the values of the fields all disappear after viewing the entry for a split second. However, clicking "Update" in Mongol on the entry actually shows the values again (I am using this as a pseudo workaround for the moment). Clicking "Cancel" or "Save" after clicking "Update" once again returns the entry to showing the fields, but not the values.

I am on Meteor 1.1.0.2
Packages:
- meteor-platform
- coffeescript
- d3
- accounts-base
- accounts-password
- iron:router
- aldeed:collection2
- ecwyne:mathjs
- fortawesome:fontawesome
- fourseven:scss
- mrt:jquery-ui
- mrt:jquery-ui-bootstrap
- numeral:numeral
- twbs:bootstrap
- u2622:persistent-session
- useraccounts:unstyled
- yogiben:admin
- msavin:mongol

**Comment by msavin** [comment]
Hmm -> any chance you are running some CSS classes directly on form elements?

**Comment by saturnaut** [comment]
That was my original thought too, but I commented out all the imports in my main SASS file, which disabled all the styles that I'm running. I also though it could be the yogiben:admin package, so I cloned my project and removed it, with no luck.

I created a new Meteor project with a few dummy collections to check out a fresh install of Mongol, and it works fine with my current version of Meteor. Went though and added the packages at the current versions I use one by one, with the same result. This tells me there is probably an issue in my project code someplace, could be a helper or some jQuery that is interfering with Mongol. The codebase is somewhat large so it will take me some time to go though it. I'm 99% sure it isn't an issue on your side.

Edit: I'll close the issue. Still haven't found the problem but I'll update when I do.
